src/providers/aws/s3-file-storage.config.ts

```diff
@@ -0,0 +1,6 @@
+export class S3FileStorageConfig{
+    s3ServiceAccessKey: string;
+    s3ServiceSecretKey: string;
+    s3ServiceRegion: string;
+    s3ServiceBucket: string;
```

Comment:
it is already s3config so no need for redundancy.

accessKey
secretKey
region
bucket